frontend/app/src/App.tsx

```diff
@@ -1525,6 +1529,20 @@ export class App extends PureComponent<Props, State> {
     this.openDeployDialog()
   }
 
+  requestFileURLs = (requestId: string, files: File[]): void => {
+    if (this.isServerConnected()) {
+      const backMsg = new BackMsg({
```

Comment:
This back message doesn't trigger a rerun, or?

Reply:
correct, it doesn't trigger a rerun